internal/lido/contracts/csmodule/nodeOperator.go

```diff
@@ -0,0 +1,127 @@
+package csmodule
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/NethermindEth/sedge/configs"
+	"github.com/NethermindEth/sedge/internal/lido/contracts"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var deployedContractAddresses = map[string]string{
+	configs.NetworkHolesky: "0x4562c3e63c2e586cD1651B958C22F88135aCAd4f",
+}
+
+/*
+NodeID :
+This function is responsible for:
+retrieving NodeOperatorID for Lido CSM node
+params :-
+network (string): The name of the network (e.g."holesky").
+rewardaddress (string): The reward address of the node operator
+returns :-
+a. *big.Int
+Node Operator ID
+b. error
+Error if any
+*/
+func NodeID(network string, rewardAddress string) (*big.Int, error) {
+	// Convert the reward address to a common.Address and check if it's zero
+	rewardAddr := common.HexToAddress(rewardAddress)
+	if rewardAddr == (common.Address{}) {
+		return nil, fmt.Errorf("invalid reward address: can't be zero address")
```

Comment:
```suggestion
		return nil, fmt.Errorf("invalid reward address")
```

Reply:
@AntiD2ta recommended to mention that a zero address is invalid, to make the error message clear.

Reply:
@stdevMac if you still think your suggestion should go through, please expand on your reasons.